metrics/prometheus_machine_test.go

```diff
@@ -25,11 +25,14 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/expfmt"
 	"github.com/stretchr/testify/assert"
+
```

Comment:
There only empty lines added in this file so you don't need to edit this file. Please remove your changes.